src/osg/Texture.cpp

```diff
@@ -1242,7 +1245,7 @@ Texture::Texture():
             _sourceType(0),
             _use_shadow_comparison(false),
             _shadow_compare_func(LEQUAL),
-            _shadow_texture_mode(LUMINANCE),
+            _shadow_texture_mode(NONE),
```

Comment:
_use_shadow_comparison being false means that the_shadow_* parameters will be ignored, there isn't any need to have the mode set to NONE.  It will also potentially break 3rd party applications and the serializers that are based on this original default. This seems to be a change for sake of change without thought for the potential consequences.

Reply:
the problem is in osg::StateSet::generateSamplerObjects:
When Sampler:: _shadow_texture_mode is setted to LUMINANCE it yields GL error